arangosh/Utils/ManagedDirectory.cpp

```diff
@@ -637,4 +637,20 @@ ssize_t ManagedDirectory::File::offset() const {
   return fileBytesRead;
 }
 
+void ManagedDirectory::File::skip(size_t count) {
+  // TODO is there a better implementation than just read count bytes?
+  // how does this work with gzip?
+  size_t const bufferSize = 4 * 1024;
+  char buffer[bufferSize]; // alloc on the heap?
+
+  while (count > 0) {
+    TRI_read_return_t bytesRead = read(buffer, std::min(bufferSize, count));
```

Comment:
It wasn't clear to me that `read(...)` here means `this->read(...)`, so I was thinking about `man 2 read` first.
Would be more explicit to write it as `this->read(...)` so there won't be any confusion.